mmdet3d/core/bbox/structures/coord_3d_mode.py

```diff
@@ -247,13 +167,13 @@ def convert_point(point, src, dst, rt_mat=None):
             rt_mat = arr.new_tensor(rt_mat)
         if rt_mat.size(1) == 4:
             extended_xyz = torch.cat(
-                [arr[:, :3], arr.new_ones(arr.size(0), 1)], dim=-1)
+                [arr[..., :3], arr.new_ones(arr.size(0), 1)], dim=-1)
```

Comment:
May update the NxK in docstring

Reply:
Cannot apply to ...xNxK, may support this in the future?